core/src/storage/impls/merkle_patricia_trie/mpt_cursor.rs

```diff
@@ -1129,9 +1158,9 @@ impl<Mpt: GetRwMpt> ReadWritePathNode<Mpt> {
                 .basic_node
                 .trie_node
                 .get_child(self.basic_node.next_child_index)
-                // Unwrap is safe. See comment below.
-                .unwrap()
-                .subtree_size as i64
+                .map(|child| child.subtree_size as i64)
```

Comment:
Better fix it in SnapshotMptNode::EMPTY_CHILD. First rename EMPTY_CHILD to NO_CHILD, then derive Default for SubtreeMerkleWithSize, and use Default::default() where EMPTY_CHILD is used in add_new_child_unchecked()